common/source/docs/common-holybro-kakutef4-mini.rst

```diff
@@ -94,8 +94,8 @@ to set the following parameters to enable support for FrSky S.PORT. It
 has built-in inverters and a diode to allow for operation from a single
 pin with no special adapters.
 
-  - :ref:`SERIAL2_PROTOCOL<SERIAL2_PROTOCOL>` 4 or 10
-  - :ref:`SERIAL2_OPTIONS<SERIAL2_OPTIONS>` 0
+  - :ref:`SERIAL1_PROTOCOL<SERIAL2_PROTOCOL>` 4 or 10
```

Comment:
good catch, but you have to change the number within the <   > also...since you are editing on the web, I will do this after I merge it...